frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -68,20 +55,22 @@ const Header: React.FC = () => {
 
   return (
     <>
-      <AppBar position="static" color="secondary">
+      <AppBar position="static" color="secondary" elevation={0}>
         <Toolbar>
           <MenuButton onClick={openDrawer} edge="start" color="primary" data-qa="menuBtn">
             <MenuIcon />
           </MenuButton>
           <Link to="/">
             <Logo />
           </Link>
-          <Divider orientation="vertical" flexItem />
-          <Title variant="h5">clutch</Title>
+          <Title>clutch</Title>
           <Box />
           <SearchField />
           <Box />
-          <UserInformation />
+          <Grid container alignItems="center" justify="flex-end">
+            <Notifications />
+            <UserInformation />
```

Comment:
the avatar is not vertically aligned

Reply:
same for clutch logo actually, slightly offset  up

Reply:
fixed both. the avatar had a bottom padding and the logo wasn't aligned in its container.
<img width="1714" alt="Screen Shot 2020-12-01 at 10 56 04 AM" src="https://user-images.githubusercontent.com/39421794/100763978-d6c7f500-33c3-11eb-91e7-3826eb8bece3.png">

Reply:
I think the avatar is supposed to have a opaque white border